2025-03-11 23:42:20,303 [INFO] USER:abhinandankumar_giri handlers.py:106:syndicate LOG: Configuration used: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\.syndicate-config-dev
2025-03-11 23:42:22,591 [WARNING] USER:abhinandankumar_giri __init__.py:190:initialize_project_state LOG: [93mConfig is set and generated but project state does not exist, seems that you've come from the previous version.[0m
2025-03-11 23:42:22,592 [WARNING] USER:abhinandankumar_giri __init__.py:193:initialize_project_state LOG: [93mGenerating project state file (.syndicate) from the existing structure...[0m
2025-03-11 23:42:22,614 [INFO] USER:abhinandankumar_giri project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\app\lambdas.
2025-03-11 23:42:22,615 [INFO] USER:abhinandankumar_giri project_state.py:400:_update_lambdas_from_path LOG: Going to add the following 'nodejs' lambda:'sns_handler' to the pending ProjectState.
2025-03-11 23:42:22,615 [INFO] USER:abhinandankumar_giri project_state.py:400:_update_lambdas_from_path LOG: Going to add the following 'nodejs' lambda:'sqs_handler' to the pending ProjectState.
2025-03-11 23:42:22,617 [INFO] USER:abhinandankumar_giri processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-11 23:42:24,238 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:42:24,238 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:42:24,637 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-11 23:42:26,259 [INFO] USER:abhinandankumar_giri handlers.py:142:test LOG: Running tests...
2025-03-11 23:42:26,260 [INFO] USER:abhinandankumar_giri handlers.py:150:test LOG: Tests not found, 'tests' folder is missing in 'C:/Users/abhinandankumar_giri/Desktop/Campus Training/aws/Aws-Lambda/task04'.
2025-03-11 23:42:26,261 [INFO] USER:abhinandankumar_giri helper.py:330:timed LOG: Stage test, elapsed time: 0:00:00.001087
2025-03-11 23:42:26,265 [INFO] USER:abhinandankumar_giri handlers.py:993:assemble LOG: Building artifacts, bundle: task04_250311.234224
2025-03-11 23:42:26,265 [INFO] USER:abhinandankumar_giri project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\jsrc\main\java\com\task04.
2025-03-11 23:42:26,266 [INFO] USER:abhinandankumar_giri project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\src\lambdas.
2025-03-11 23:42:26,266 [INFO] USER:abhinandankumar_giri project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\app\lambdas.
2025-03-11 23:42:26,267 [INFO] USER:abhinandankumar_giri project_state.py:400:_update_lambdas_from_path LOG: Going to add the following 'nodejs' lambda:'sns_handler' to the pending ProjectState.
2025-03-11 23:42:26,267 [INFO] USER:abhinandankumar_giri project_state.py:400:_update_lambdas_from_path LOG: Going to add the following 'nodejs' lambda:'sqs_handler' to the pending ProjectState.
2025-03-11 23:42:26,267 [INFO] USER:abhinandankumar_giri project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\dnapp\lambdas.
2025-03-11 23:42:26,267 [INFO] USER:abhinandankumar_giri project_state.py:430:_update_bpm_resources_from_path LOG: Going to resolve any build project mapping dependent resources from a given path: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\swagger_src.
2025-03-11 23:42:26,267 [INFO] USER:abhinandankumar_giri project_state.py:430:_update_bpm_resources_from_path LOG: Going to resolve any build project mapping dependent resources from a given path: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\appsync_src.
2025-03-11 23:42:26,271 [INFO] USER:abhinandankumar_giri handlers.py:821:assemble_node LOG: Command assemble node: project_path: app 
2025-03-11 23:42:26,272 [INFO] USER:abhinandankumar_giri nodejs.py:58:assemble_node_lambdas LOG: Going to package lambdas starting by path C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\app
2025-03-11 23:42:26,273 [INFO] USER:abhinandankumar_giri nodejs.py:64:assemble_node_lambdas LOG: Going to build artifact in: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\app\lambdas\sns_handler
2025-03-11 23:42:26,274 [INFO] USER:abhinandankumar_giri nodejs.py:64:assemble_node_lambdas LOG: Going to build artifact in: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\app\lambdas\sqs_handler
2025-03-11 23:42:26,807 [INFO] USER:abhinandankumar_giri nodejs.py:123:install_requirements LOG: Artifacts path: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\.syndicate-config-dev\bundles\task04_250311.234224\sqs_handler-1.0
2025-03-11 23:42:26,808 [INFO] USER:abhinandankumar_giri nodejs.py:123:install_requirements LOG: Artifacts path: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\.syndicate-config-dev\bundles\task04_250311.234224\sns_handler-1.0
2025-03-11 23:42:27,782 [INFO] USER:abhinandankumar_giri helper.py:107:execute_command_by_path LOG: Running the command "npm install"

up to date, audited 1 package in 527ms

found 0 vulnerabilities


2025-03-11 23:42:27,782 [INFO] USER:abhinandankumar_giri nodejs.py:142:install_requirements LOG: No dependencies folder - nothing to copy.
2025-03-11 23:42:27,784 [INFO] USER:abhinandankumar_giri helper.py:107:execute_command_by_path LOG: Running the command "npm install"

up to date, audited 1 package in 537ms

found 0 vulnerabilities


2025-03-11 23:42:27,784 [INFO] USER:abhinandankumar_giri nodejs.py:142:install_requirements LOG: No dependencies folder - nothing to copy.
2025-03-11 23:42:27,809 [INFO] USER:abhinandankumar_giri handlers.py:830:assemble_node LOG: NodeJS artifacts were prepared successfully.
2025-03-11 23:42:27,810 [INFO] USER:abhinandankumar_giri helper.py:330:timed LOG: Stage assemble_node, elapsed time: 0:00:01.538386
2025-03-11 23:42:27,814 [INFO] USER:abhinandankumar_giri helper.py:330:timed LOG: Stage assemble, elapsed time: 0:00:01.549829
2025-03-11 23:42:27,819 [INFO] USER:abhinandankumar_giri handlers.py:1037:package_meta LOG: Package meta, bundle: task04_250311.234224
2025-03-11 23:42:27,819 [INFO] USER:abhinandankumar_giri meta_processor.py:513:create_meta LOG: Bundle path: C:/Users/abhinandankumar_giri/Desktop/Campus Training/aws/Aws-Lambda/task04/bundles/task04_250311.234224
2025-03-11 23:42:27,823 [INFO] USER:abhinandankumar_giri helper.py:292:write_content_to_file LOG: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\.syndicate-config-dev/bundles/task04_250311.234224\build_meta.json file was created.
2025-03-11 23:42:27,823 [INFO] USER:abhinandankumar_giri handlers.py:1040:package_meta LOG: Meta was configured successfully.
2025-03-11 23:42:27,823 [INFO] USER:abhinandankumar_giri helper.py:330:timed LOG: Stage package_meta, elapsed time: 0:00:00.003973
2025-03-11 23:42:28,021 [INFO] USER:abhinandankumar_giri handlers.py:1083:upload LOG: Upload bundle: task04_250311.234224
2025-03-11 23:42:28,427 [INFO] USER:abhinandankumar_giri bundle_processor.py:232:upload_bundle_to_s3 LOG: Bundle was validated successfully
2025-03-11 23:42:29,870 [INFO] USER:abhinandankumar_giri handlers.py:1090:upload LOG: Bundle was uploaded successfully
2025-03-11 23:42:29,875 [INFO] USER:abhinandankumar_giri helper.py:330:timed LOG: Stage upload, elapsed time: 0:00:02.048494
2025-03-11 23:42:29,885 [INFO] USER:abhinandankumar_giri helper.py:330:timed LOG: Stage build, elapsed time: 0:00:05.246980
2025-03-11 23:42:36,143 [INFO] USER:abhinandankumar_giri handlers.py:106:syndicate LOG: Configuration used: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\.syndicate-config-dev
2025-03-11 23:42:38,416 [INFO] USER:abhinandankumar_giri processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-11 23:42:40,105 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:42:40,105 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:42:40,507 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-11 23:42:40,926 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:42:40,926 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:42:41,322 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-11 23:42:41,323 [INFO] USER:abhinandankumar_giri helper.py:233:resolve_default_value LOG: Resolved value of deploy_name: task04
2025-03-11 23:42:41,729 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:42:41,730 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:42:42,141 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-11 23:42:42,141 [INFO] USER:abhinandankumar_giri helper.py:233:resolve_default_value LOG: Resolved value of bundle_name: task04_250311.234224
2025-03-11 23:42:42,568 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:42:42,568 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:42:43,069 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-11 23:42:43,069 [ERROR] USER:abhinandankumar_giri decorators.py:136:wrapper LOG: The project modification_lock is locked.
2025-03-11 23:42:43,073 [ERROR] USER:abhinandankumar_giri decorators.py:137:wrapper LOG: Traceback (most recent call last):
  File "C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\syndicate_venv\Lib\site-packages\syndicate\core\decorators.py", line 134, in wrapper
    return_code = func(*args, **kwargs)
  File "C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\syndicate_venv\Lib\site-packages\syndicate\core\helper.py", line 305, in wrapper
    raise AssertionError(f'The project {lock_type} is locked.')
AssertionError: The project modification_lock is locked.
Traceback (most recent call last):
  File "C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\syndicate_venv\Lib\site-packages\syndicate\core\decorators.py", line 134, in wrapper
    return_code = func(*args, **kwargs)
  File "C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\syndicate_venv\Lib\site-packages\syndicate\core\helper.py", line 305, in wrapper
    raise AssertionError(f'The project {lock_type} is locked.')
AssertionError: The project modification_lock is locked.
2025-03-11 23:44:11,100 [INFO] USER:abhinandankumar_giri handlers.py:106:syndicate LOG: Configuration used: C:\Users\abhinandankumar_giri\Desktop\Campus Training\aws\Aws-Lambda\task04\.syndicate-config-dev
2025-03-11 23:44:13,385 [INFO] USER:abhinandankumar_giri processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-11 23:44:14,928 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:44:14,929 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:44:15,433 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-11 23:44:15,849 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:44:15,850 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:44:16,354 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-11 23:44:16,355 [INFO] USER:abhinandankumar_giri helper.py:233:resolve_default_value LOG: Resolved value of deploy_name: task04
2025-03-11 23:44:16,771 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:44:16,771 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:44:17,277 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-11 23:44:17,278 [INFO] USER:abhinandankumar_giri helper.py:233:resolve_default_value LOG: Resolved value of bundle_name: task04_250311.234224
2025-03-11 23:44:17,694 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:44:17,695 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:44:18,197 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-11 23:44:18,617 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:44:18,617 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:44:19,119 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-11 23:44:21,439 [INFO] USER:abhinandankumar_giri deployment_processor.py:466:create_deployment_resources LOG: Prefixes and suffixes of any resource names have been resolved
2025-03-11 23:44:21,440 [INFO] USER:abhinandankumar_giri deployment_processor.py:489:create_deployment_resources LOG: Going to deploy AWS resources
2025-03-11 23:44:21,755 [INFO] USER:abhinandankumar_giri deployment_processor.py:74:_process_resources LOG: Processing iam_policy resources
2025-03-11 23:44:23,325 [WARNING] USER:abhinandankumar_giri iam_resource.py:115:_create_policy_from_meta LOG: IAM policy cmtr-3bfb6220-lambda-basic-execution exists.
2025-03-11 23:44:23,627 [INFO] USER:abhinandankumar_giri deployment_processor.py:74:_process_resources LOG: Processing iam_role resources
2025-03-11 23:44:25,169 [WARNING] USER:abhinandankumar_giri iam_resource.py:141:_create_role_from_meta LOG: IAM role cmtr-3bfb6220-sqs_handler-role exists.
2025-03-11 23:44:25,177 [ERROR] USER:abhinandankumar_giri deployment_processor.py:387:process_response LOG: Caused by resource named cmtr-3bfb6220-sns_handler-role. Parameter validation failed:
Invalid length for parameter PermissionsBoundary, value: 9, valid min length: 20
2025-03-11 23:44:25,178 [INFO] USER:abhinandankumar_giri deployment_processor.py:74:_process_resources LOG: Processing sns_topic resources
2025-03-11 23:44:26,542 [ERROR] USER:abhinandankumar_giri deployment_processor.py:387:process_response LOG: Caused by resource named cmtr-3bfb6220-lambda_topic. An error occurred (AuthorizationError) when calling the ListTopics operation: User: arn:aws:sts::890742610830:assumed-role/cmtr_syndicate_student_programmatic_accessv2/3bfb6220 is not authorized to perform: SNS:ListTopics on resource: arn:aws:sns:eu-central-1:890742610830:* with an explicit deny in a service control policy
2025-03-11 23:44:26,543 [INFO] USER:abhinandankumar_giri deployment_processor.py:74:_process_resources LOG: Processing sqs_queue resources
2025-03-11 23:44:27,822 [WARNING] USER:abhinandankumar_giri sqs_resource.py:104:_create_sqs_queue_from_meta LOG: SQS queue cmtr-3bfb6220-async_queue exists.
2025-03-11 23:44:28,027 [INFO] USER:abhinandankumar_giri deployment_processor.py:88:_process_resources LOG: Processing lambda resources
2025-03-11 23:44:30,633 [WARNING] USER:abhinandankumar_giri lambda_resource.py:313:_create_lambda_from_meta LOG: cmtr-3bfb6220-sqs_handler lambda exists.
2025-03-11 23:44:31,858 [ERROR] USER:abhinandankumar_giri deployment_processor.py:387:process_response LOG: Caused by resource named cmtr-3bfb6220-sns_handler. Role cmtr-3bfb6220-sns_handler-role does not exist; Lambda cmtr-3bfb6220-sns_handler failed to be configured.
2025-03-11 23:44:32,354 [INFO] USER:abhinandankumar_giri deployment_processor.py:527:create_deployment_resources LOG: Going to apply post deployment tags
2025-03-11 23:44:32,361 [INFO] USER:abhinandankumar_giri group_tagging_api_resource.py:135:apply_tags LOG: Tags were successfully applied
2025-03-11 23:44:32,361 [INFO] USER:abhinandankumar_giri deployment_processor.py:530:create_deployment_resources LOG: Going to create deploy output
2025-03-11 23:44:33,971 [INFO] USER:abhinandankumar_giri bundle_processor.py:71:create_deploy_output LOG: Output file with name task04_250311.234224/outputs/task04_failed.json has been created
2025-03-11 23:44:33,971 [INFO] USER:abhinandankumar_giri deployment_processor.py:560:create_deployment_resources LOG: Deploy output for task04 was created.
2025-03-11 23:44:33,972 [WARNING] USER:abhinandankumar_giri handlers.py:344:deploy LOG: Backend resources were deployed with errors. See deploy output file.
2025-03-11 23:44:33,972 [INFO] USER:abhinandankumar_giri helper.py:330:timed LOG: Stage deploy, elapsed time: 0:00:14.853502
2025-03-11 23:44:34,384 [INFO] USER:abhinandankumar_giri project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-11 23:44:34,384 [INFO] USER:abhinandankumar_giri project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-11 23:44:34,786 [INFO] USER:abhinandankumar_giri sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
